src/app/components/AccountInfo/NodeUriModal.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Modal } from 'antd';
+import { getNodeInfo } from 'modules/node/actions';
+import QRCode from 'qrcode.react';
+import Copy from 'components/Copy';
+import { AppState } from 'store/reducers';
+import './DepositModal.less';
+
+interface StateProps {
+  node: AppState['node']['nodeInfo'];
+}
+
+interface DispatchProps {
+    getNodeInfo: typeof getNodeInfo;
```

Comment:
getNodeInfo is never used in this component

Reply:
Dang, I'm slippin. Forgot to put {} when there is no DispatchToProps. Fixed in d02a89f.

Reply:
Dang, I'm slippin. Forgot to put {} when there is no DispatchToProps. Fixed in